### PR TITLE
Update supported releas

### PR DESCRIPTION
**Pull request by brian-avery**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by jacob-delgado — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Review by jacob-delgado — APPROVED** [review]

File: content/en/news/security/istio-security-2021-003/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description:
 cves: [CVE-2021-28683, CVE-2021-28682, CVE-2021-29258]
 cvss: "7.5"
 vector: "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
-releases: ["1.8.5", "1.9.3"]
+releases: ["1.8.4 and below", "1.9.0 to 1.9.2"]
```
Comment (ericvn) [inline content/en/news/security/istio-security-2021-003/index.md:8]: Interesting that you have this two ways. It seems we standardized on the `to`.

Comment (brian-avery) [inline content/en/news/security/istio-security-2021-003/index.md:8]:
```suggestion
releases: ["All releases prior to 1.8.5", "1.9.0 to 1.9.2"]
```

**Comment by brian-avery** [comment]
Thanks @ericvn

**Pull request merged**